content/learning-how-to-teach-coders/quizzes/_index.md

```diff
@@ -1,5 +1,45 @@
 ---
-_db_id: 649
 content_type: topic
 title: Quizzes
----
\ No newline at end of file
+---
+
+## What The Sessions Should Be About
+
+<!-- This type of session involves the leader and follower working through some kind of Q&A type quiz. The topic could be very basic or something more advanced. -->
+
+The session should be about a junior staff members or jtl giving the recruits some questions, quizzes on various topics to work through and see if recruits have a strong and good understanding on the basics foundational skills.
```

Comment:
What if there is another code school making use of our open source content? Will they understand this session?

Should we not just say that there is a leader and a follower and the leader needs to give the follower some questions etc? Why superglue this to the Umuzi context?